test/test_jit_fuser_te.py

```diff
@@ -1146,24 +1146,27 @@ def rand(dtype, device="cuda"):
                 return torch.randint(0, 100, shape, dtype=dtype, device=device)
             raise RuntimeError("Unhandled dtype")
 
-        dtypes = [
+        supported_dtypes = [
             torch.int8,
             torch.uint8,
             torch.int16,
             torch.int32,
             torch.int64,
             # torch.float16,
-            # torch.bfloat16,
```

Comment:
Is it intentional that `torch.float16` remains commented out?

Reply:
Yeah, some fixes for float16 are in the works.  This should probably be a TODO but I expect it to land in a day or so.